app/services/idv/actions/ipp/redo_address_action.rb

```diff
@@ -0,0 +1,11 @@
+module Idv
+  module Actions
+    module Ipp
+      class RedoAddressAction < Idv::Steps::DocAuthBaseStep
```

Comment:
This action takes advantage of the flow state machine which tracks the user's progress through various flows of the application -- in this case the in-person proofing flow. When the user clicks the change address button this action marks the address step as incomplete which redirects the user back to the address page where they can update their information

I am essentially just copying the existing pattern used by the [RedoSsnAction](https://github.com/18F/identity-idp/blob/main/app/services/idv/actions/redo_ssn_action.rb). I'm not actually sure if we need to inherit from DocAuthBaseStep here though -- do you know if there are metrics or error handling that would break if I don't @zachmargolis?

Reply:
I don't know what would break, but I would probably do the same and copy existing